scene/resources/particle_process_material.cpp

```diff
@@ -2209,6 +2235,8 @@ void ParticleProcessMaterial::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "sub_emitter_amount_at_collision", PROPERTY_HINT_RANGE, "1,32,1"), "set_sub_emitter_amount_at_collision", "get_sub_emitter_amount_at_collision");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "sub_emitter_keep_velocity"), "set_sub_emitter_keep_velocity", "get_sub_emitter_keep_velocity");
 
+	ADD_SIGNAL(MethodInfo("emission_shape_changed"));
```

Comment:
Let's remove this so we don't have to emit the signal at runtime

Reply:
Removed 👍🏻